README.md

```diff
@@ -252,20 +252,24 @@ The following generic methods are added to `nemo.view`
 
 `@returns {Promise}` resolves to true or rejected
 
-#### _wait(locatorString[, timeout])
+#### _wait(locatorString[, timeout, [, msg]])
```

Comment:
Almost. Consider the brackets denote an optional feature. Thus, the comma before msg must be inside of the brackets, not outside of:
`_wait(locatorString[, timeout[, msg]])`

This is because the comma before msg is only necessary when you actually include a msg parameter